tests/test_lando.py

```diff
@@ -0,0 +1,122 @@
+import unittest
+from unittest import mock
+
+from libmozdata.lando import LandoWarnings
+
+MOCK_LANDO_API_URL = "http://api.lando.test/"
+MOCK_LANDO_TOKEN = "Some Test Token"
+
+MOCK_DIFF_ID = 0
+MOCK_REV_ID = 0
+MOCK_GROUP = "LINT"
+
+MOCK_WARNINGS_ID = 100
+
+MOCK_WARNING_MESSAGE = "There are still warnings"
+
+MOCK_WARNINGS_DATA = [
+    {
+        "data": MOCK_WARNING_MESSAGE,
+        "diff_id": MOCK_DIFF_ID,
+        "rev_id": MOCK_REV_ID,
+        "group": MOCK_GROUP,
+        "id": MOCK_WARNINGS_ID,
+    }
+]
+
+
+def mocked_get_warnings(*args, **kwargs):
+    class MockResponse:
+        def __init__(self, json_data, status_code):
+            self.json_data = json_data
+            self.status_code = status_code
+
+        def json(self):
+            return self.json_data
+
+    # Test URL passed to requests.get
+    assert args[0] == MOCK_LANDO_API_URL + "/diff_warnings"
+
+    # Test the `params` passed to requests.get
+    assert kwargs["params"] == {
+        "revision_id": MOCK_REV_ID,
+        "diff_id": MOCK_DIFF_ID,
+        "group": MOCK_GROUP,
+    }
+
+    # Test the `headers`
+    assert kwargs["headers"] == {"X-Phabricator-API-Key": MOCK_LANDO_TOKEN}
+
+    return MockResponse(MOCK_WARNINGS_DATA, 200)
+
+
+def mocked_add_warning(*args, **kwargs):
+    class MockResponse:
+        def __init__(self, status_code):
+            self.status_code = status_code
+
+    # Test URL passed to requests.post
+    assert args[0] == MOCK_LANDO_API_URL + "/diff_warnings"
+
+    # Test the `params` passed to requests.get
+    assert kwargs["json"] == {
+        "revision_id": MOCK_REV_ID,
+        "diff_id": MOCK_DIFF_ID,
+        "group": MOCK_GROUP,
+        "data": {"message": MOCK_WARNING_MESSAGE},
+    }
+
+    # Test the `headers`
+    assert kwargs["headers"] == {"X-Phabricator-API-Key": MOCK_LANDO_TOKEN}
+
+    return MockResponse(201)
+
+
+def mocked_del_warnings(*args, **kwargs):
+    class MockResponse:
+        def __init__(self, status_code):
+            self.status_code = status_code
+
+    # Test URL passed to requests.del
+    assert args[0] == MOCK_LANDO_API_URL + "/diff_warnings/" + str(MOCK_WARNINGS_ID)
+
+    # Test the `headers`
+    assert kwargs["headers"] == {"X-Phabricator-API-Key": MOCK_LANDO_TOKEN}
+
+    return MockResponse(200)
+
+
+class Test_LandoWarnings(unittest.TestCase):
+    @mock.patch("requests.get", side_effect=mocked_get_warnings)
```

Comment:
We are using `responses` to mock responses in the other tests in this library, can you do the same here?